### PR TITLE
Luna983/data/update/fig a2

### PR DESCRIPTION
**Pull request by luna983**

### CODE REVIEW

File: codes/data/china/collate_data.py [hunk 1]
```diff
@@ -376,9 +376,14 @@ def log_interpolate(array):
...
+# https://www.medrxiv.org/content/10.1101/2020.03.23.20041319v1.full.pdf
 df.loc[:, 'testing_regime'] = (
-    (df['date'] >= pd.Timestamp('2020-02-13')).astype(int) +
-    (df['date'] >= pd.Timestamp('2020-02-20')).astype(int)
+    (df['date'] > pd.Timestamp('2020-01-17')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-01-27')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-02-05')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-02-12')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-02-19')).astype(int) +
+    (df['date'] > pd.Timestamp('2020-03-04')).astype(int)
```
Comment (bolliger32) [inline codes/data/china/collate_data.py:386]: waiting on response to @jeanettelt's comments from #120

File: codes/plotting/figA2.py [hunk 2]
```diff
@@ -21,16 +22,17 @@ def main():
 
     # validate with JHU
     url = (
-        'https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/'
-        'csse_covid_19_time_series/time_series_19-covid-Confirmed.csv')
+        'https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/'
```
Comment (bolliger32) [inline codes/plotting/figA2.py:25]: We already have the data downloaded from @kendonB's script right? Is there a reason we don't just pull from that csv?

File: codes/plotting/figA2.py [hunk 3]
```diff
@@ -62,23 +65,19 @@ def main():
...
-        ax_i.minorticks_off()
     fig.tight_layout()
     fig.savefig(out_dir / 'figA2-1.pdf')
 
-
     df_kor = pd.read_csv(cutil.DATA_INTERIM / 'korea' / 'KOR_JHU_data_comparison.csv')
-
-    df_kor = df_kor.iloc[0:56, :].copy()
-
+    # df_kor = df_kor.iloc[0:56, :].copy()
```
Comment (bolliger32) [inline codes/plotting/figA2.py:78]: Should we just delete this line?

**Comment by bolliger32** [comment]
pipeline error is due to known issue in USA, which is addressed by #122 , so Im not worried about that.

File: codes/plotting/figA2.py (hunk 2 above)
Comment (luna983) [reply codes/plotting/figA2.py:25]: I wanted to compare against the raw data (Kenny implemented some additional cleaning steps, which is great for analysis) but this exercise is a bit different

File: codes/data/china/collate_data.py (hunk 1 above)
Comment (luna983) [reply codes/data/china/collate_data.py:386]: we sorted this out between ourselves and will continue to use these dates; there are minor differences between these and the medrxiv paper, but these correspond to data jumps and news reports better

File: codes/plotting/figA2.py (hunk 3 above)
Comment (luna983) [reply codes/plotting/figA2.py]: yup

File: codes/data/china/collate_data.py (hunk 1 above)
Comment (luna983) [reply codes/data/china/collate_data.py:386]: Just responded to that

**Pull request merged**